content/en/apps/reference/app-settings/transitions.md

```diff
@@ -671,4 +672,63 @@ If this configuration is not set then the message defaults to what is set in the
     ]
   }
 ]
-```
\ No newline at end of file
+```
+
+### create_user_for_contacts
+
+An existing offline user can be replaced on a device so that a new user can use that device without needing to immediately sync with the server. _Available since 3.17.x._
+
+#### Example scenario
+
+Imagine a CHW is leaving the program, and the CHW's device is returned to their supervisor. The supervisor wants to transfer the device to a new CHW immediately without having the device online to sync with the server.
+
+To do this, when the `create_user_for_contacts` transition is enabled, the supervisor would submit a configured user replacement form for the original user's contact on the device. This form can create a new contact for the new CHW and will trigger a client-side transition to mark the original contact as replaced. After that, the supervisor can give the device to the new CHW, and they can begin using it.
+
+Subsequent reports submitted on the device by the new CHW will be associated with the new contact. When the device is eventually able to synchronize with the server, it will be automatically logged out (as it was actually still logged in as the original user). A server-side transition will be triggered to inactivate the original user and to create a new user for the new CHW. An SMS message containing a token login link will be sent to the new CHW allowing them to login as their proper user. 
```

Comment:
Can the previous account still be used to log in? I think we should cover this in the documentation.